release/models/platform/openconfig-platform.yang

```diff
@@ -726,6 +731,83 @@ revision "2024-05-29" {
 
   }
 
+  grouping platform-component-install-state {
+    description
+      "Install state data for platform components";
+
+    container install {
+      description
+        "Top-level container for component install state data";
+
+      leaf new-version {
+        type string;
+        description
+          "This is the version that is available on the component,
+          but the install is not complete or has not yet started.";
+      }
+
+      leaf new-version-service-impacting {
+        type boolean;
+        description
+          "A value of true indicates that installing would
+          be a service impacting event.";
+      }
+
+      leaf status {
+        type identityref {
+          base oc-platform-types:OPENCONFIG_INSTALL_STATUS;
+        }
+        description
+          "The status of the install.";
+      }
+
+      leaf step {
+        type string;
+        description
+          "Description of the current in-progress step of the install.";
+      }
+
+      leaf step-percent-complete {
+        type oc-types:percentage;
+        description
+          "The percent complete for the in-progress step.";
+      }
+
+      leaf total-percent-complete {
+        type oc-types:percentage;
+        description
+          "The percent complete for the entire install process.";
+      }
+
+      leaf start-timestamp {
+        type oc-types:timeticks64;
+        description
+          "The timestamp when the install started.";
+      }
+
+      leaf duration {
```

Comment:
It feels like there is some overlap/duplication in having a start-timestamp, stop-timestamp, and duration. And would duration constantly update every second? (i.e. be very noisy if subscriber to ON_CHANGE)? I'd propose we drop the duration leaf here.